Schmidt/template/README.md

```diff
@@ -1,13 +1,13 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+
+factorial is a function that returns something important.
```

Comment:
What is a factorial? Could you maybe include the mathematical formulate, eg:

factorial(10) returns 10!

Reply:
Cool.